examples/test_error.py

```diff
@@ -101,17 +101,14 @@
     # }}}
 
     # {{{Normalization
-    frq_slice = integrate_limits(select_pathway(s, signal_pathway),
-            convolve_method='Gaussian')
+    frq_slice = integrate_limits(select_pathway(s, signal_pathway))
```

Comment:
it's confusing to have the `integrate_w_error` to select the non-default option for `integrate_limits` by default, so I modify the former instead of what you did here.

Reply:
`gaussian` is the default - that was more of a trouble shooting thing that we left in, so it is fine to remove